Test/Classes/TestTextUtils.cs

```diff
@@ -67,8 +393,66 @@ public void StripDateFromNameSimilarDate()
         {
             DateTime date = new DateTime(2009, 02, 13);
 
-            Assert.Equal("Test", TextUtils.StripDateFromName("Test 16/02/2009", date));
-            Assert.Equal("Test", TextUtils.StripDateFromName("Test 11/02/2009", date));
+            try
+            {
+                // older than date by >9 days
+                // DATE should be left in the name
+                Assert.Equal("Test 01/02/2009", TextUtils.StripDateFromName("Test 01/02/2009", date));
+                Assert.Equal("01/02/2009 Test", TextUtils.StripDateFromName("01/02/2009 Test", date));
+                Assert.Equal("Test 02/02/2009 Test", TextUtils.StripDateFromName("Test 02/02/2009 Test", date));
+                Assert.Equal("Test 02/02/2009", TextUtils.StripDateFromName("Test 02/02/2009", date));
+                Assert.Equal("01/02/2009 Test", TextUtils.StripDateFromName("01/02/2009 Test", date));
+                Assert.Equal("Test 02/02/2009 Test", TextUtils.StripDateFromName("Test 02/02/2009 Test", date));
```

Comment:
I'd say that this block where the date should be left should be moved into `StripDateFromNameIgnoreDifferent`.